### PR TITLE
conductor: add genesis sequencer block height to config and env vars

### PR DESCRIPTION
**Pull request by sambukowski**
## Summary
Add an env var that specifies which sequencer block (using the block height) the genesis block of the rollup was in.

## Background
This is a requirement for adding sync functionality to the Conductor.

## Changes
- Added `ASTRIA_CONDUCTOR_GENESIS_SEQUENCER_BLOCK_HEIGHT` env var and `genesis_sequencer_block_height` to Conductor config.

## Testing
This should pass all existing tests.

## Related Issues
#439

### CODE REVIEW

**Review by joroshiba — APPROVED** [review]
Might want to bike shed the config name later but this is fine for now.

**Review by steezeburger — APPROVED** [review]

File: crates/astria-conductor/local.env.example [hunk 1]
```diff
@@ -33,3 +33,6 @@ ASTRIA_CONDUCTOR_SEQUENCER_URL="ws://127.0.0.1:26657"
 # true: skips empty blocks
 # false: executes empty blocks
 ASTRIA_CONDUCTOR_DISABLE_EMPTY_BLOCK_EXECUTION=false
+
+# the sequencer block height that the rollup genesis block was in
```
Comment (noot) [inline crates/astria-conductor/local.env.example:37]:
```suggestion
# the sequencer block height that the rollup's first block was in
```

File: crates/astria-conductor/local.env.example [hunk 2]
```diff
@@ -33,3 +33,6 @@ ASTRIA_CONDUCTOR_SEQUENCER_URL="ws://127.0.0.1:26657"
...
+ASTRIA_CONDUCTOR_GENESIS_SEQUENCER_BLOCK_HEIGHT=0
```
Comment (noot) [inline crates/astria-conductor/local.env.example:38]:
```suggestion
ASTRIA_CONDUCTOR_INIITIAL_SEQUENCER_BLOCK_HEIGHT=0
```

File: crates/astria-conductor/local.env.example (hunk 1 above)
Comment (noot) [reply crates/astria-conductor/local.env.example:37]: the genesis block of the rollup is determined by the genesis state/file - the first block that uses the sequencer would be block 1

**Pull request merged**